test/libsyncengine/test.cpp

```diff
@@ -40,7 +40,7 @@
 #include "requests/testexclusiontemplatecache.h"
 
 namespace KDC {
-CPPUNIT_TEST_SUITE_REGISTRATION(TestExclusionTemplateCache);
+/* CPPUNIT_TEST_SUITE_REGISTRATION(TestExclusionTemplateCache);
```

Comment:
Do not commit changes on this file (except if you add more tests)

Reply:
[95da3fb](https://github.com/Infomaniak/desktop-kDrive/pull/139/commits/95da3fb6f710a14e43dba25d6de463d4c6d0a462)